### DISCUSSION TITLE
Suggestion - Close Issues/PR's actions

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#4568] [Ideas]
It would be ideal to make an effort and check all the 304 issues that remain open at this time, keeping only the ones that should remain in our attention. At least in the last 6 months I have read these sections several times and I can say that they are quite "hanging".

The same action should be done with PR's older than 1 year.

**Comment by fballiano** [comment]
totally agree, I was chatting with @colinmollenhour about that some time ago.

**Comment by addison74** [comment]
Good. This could be easily solved by leaving a note for maintainers in every issue/PR we analyze. I estimate at least 20% ballast.

**Comment by fballiano** [comment]
Also, the github's issue system is being used for support request which is simply unbearable, I've no idea how we could avoid that but we can't spend time on those cause we already don't have enough for everything else.

**Comment by Flyingmana** [comment]
As for now, we still have very few actual support requests in our Issues. We could/should introduce a new Tag for them, to keep a better eye on them and have a way to filter them.
And then we still have some discussion Tickets open as well, (like this one here) which is probably also better suited as a discussion.
But thats in the end also just a small part.

I was also thinking about this Topic after it was mentioned on Twitter (and also in the past mentioned more then once)
To get a better View of how the state actually is, and how the numbers evolve over time I found this usefull tool: https://cauldron.io/project/4125?from_date=2015-01-01&to_date=2021-05-16&tab=overview

For a short summary I would say, most Issues/PRs are actually from the last year, but we are also solving Issues/PRs in an increasing number for a few months now.

The biggest risk I see here is what also happened in M2. People stopping to submit Issues/Contributions, because they were closed anyway in the end. Thats also the reason I stopped most of my contributions there, because most of them got closed because of age, inactivity and other reasons.

So much for the current State.
To tackle this there are two things for me.
1. I would like to see a few more working on this on our Discord. I post there from time to time PRs which I think would be easy to Review, or where I just need a bit of feedback from someone else.
It would help to tackle this in a more organized way
2. I want to handle this this Issue "Number of Open Issues/PRs" from the point of view of a Product. So not only focusing on the pure number of open PRs/Issues, but understanding what Issues they cause for the contributors, what are the expectations, what is seen as normal/acceptable for this number.

**Comment by luigifab** [comment]
I suggest to prepare to close: (366), 411, 470, 554, 1053, 1217, 1234, (1369?), (1415?), 1482, 1574.

With a link to https://www.openmage.org/magento-lts/install.html
But where is the unique good way howto to install OpenMage? For example:

Install:
- you need PHP 7.0+ with curl, gd, intl, json, mbstring, mysql, soap...
- then create a new directory
- create a composer.json file
- put this into the composer file
- run `composer install (--ignore-platform-reqs)`

Update:
- run `composer update (--ignore-platform-reqs)`

Migrate from Magento 1:
- ....

Is there an howto somewhere how to migrate from Magento 1 to OpenMage 19/20 with composer?

**Comment by addison74** [comment]
19 PR's are with "Review Needed" label since April the 2nd 2019. If the developers are not responding and nobody can solve the issue I suggest closing them.

**Comment by Flyingmana** [comment]
lets look directly at 366
Its a clear requirement, one which makes sense to have documented. Still we closed it now without any documentation covering it.

Not saying anything about the others. But by closing Tickets with valid requirements we basically lose this on our todo, and signaling the people involved, that we dont want this. at least thats the message which arrived to me for this specific one.

1053 and 1217 are documentation issues. As I understand, you want to close them without really documenting these cases by saying "We have one way to install, follow it or handle the problems yourself"?

**Comment by colinmollenhour** [comment]
Sorry, @Flyingmana it wasn't clear what requirement was remaining and the OP posted in response to your answer indicating that the solution was found and was successful. There are also other issues remaining open that cover the same topic and are less obsolete. Please feel free to reopen if you still intend to take action on that issue.

**Comment by Flyingmana** [comment]
its... very low on my list of things, currently I dont thin I will get to write any documentation in the next 1-2 years...

**Comment by addison74** [comment]
I close this issue. We can continue the discussions here #1654.
